pom.xml

```diff
@@ -121,10 +121,9 @@
     <gson.version>2.8.9</gson.version>
     <gson-extras.version>0.2.2</gson-extras.version>
     <org-json.version>20240205</org-json.version>
-    <jackson.version>2.12.7.20240502</jackson.version>
-    <jetty.version>9.4.52.v20230823</jetty.version>
-    <jakarta.activation.version>1.2.2</jakarta.activation.version>
-    <jakarta.xml.bind.version>2.3.3</jakarta.xml.bind.version>
+    <jackson.version>2.17.3</jackson.version>
```

Comment:
jackson 2.18.3 is better - there is at least 1 nasty bug in the jackson 2.17 parser

Reply:
I have adjusted it.